React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -195,19 +208,25 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
   // FIXME: Commented props are not supported yet.
   MAP_SCROLL_VIEW_PROP(alwaysBounceHorizontal);
   MAP_SCROLL_VIEW_PROP(alwaysBounceVertical);
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
   MAP_SCROLL_VIEW_PROP(bounces);
   MAP_SCROLL_VIEW_PROP(bouncesZoom);
   MAP_SCROLL_VIEW_PROP(canCancelContentTouches);
+#endif // ]TODO(macOS GH#774)
```

Comment:
For iOS diff tags, we prefer to leave them without the [ or ] brackets. This is to show this is not a block of code we added, but instead two lines of code we added.

Reply:
> This is to show this is not a block of code we added, but instead two lines of code we added.

To confirm, we only add the `[ ]` brackets if we are adding a new block of code? For lines that are merely disabled on macOS, we can omit the brackets?

Reply:
Yes